deploy/kubernetes/che-devfile-registry/templates/deployment.yaml

```diff
@@ -10,15 +10,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  labels:
-    app: che-devfile-registry
   name: che-devfile-registry
+  labels:
+    app: che
```

Comment:
Why do we need app? Should not it be replaced with `app.kubernetes.io/name: che`?

Reply:
We use these labels on chectl side to find a corresponding deployment.